app/javascript/utilities/podcastPlayback.js

```diff
@@ -51,29 +51,26 @@ function initializePodcastPlayback() {
       volume: 1,
       duration: 1,
       updated: new Date().getTime(),
-      windowName: window.name,
+      playbackName: audio?.dataset?.episode,
```

Comment:
This change fixes a bug that appears when doing play/pause on podcast episodes (record spinning/not spinning)